test_data/crontab

```diff
@@ -0,0 +1 @@
+* * * * * /usr/bin/echo "test"
```

Comment:
use a unique value for each piece here and verify things get set correctly in the test